library/std/src/sys/windows/c.rs

```diff
@@ -1009,61 +960,33 @@ extern "system" {
         lpNumberOfBytesTransferred: LPDWORD,
         bWait: BOOL,
     ) -> BOOL;
-    pub fn select(
-        nfds: c_int,
-        readfds: *mut fd_set,
-        writefds: *mut fd_set,
-        exceptfds: *mut fd_set,
-        timeout: *const timeval,
-    ) -> c_int;
-
-    // >= Vista / Server 2008
-    // https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createsymboliclinkw
```

Comment:
Is there any reason why u deleted this comments (and lower)?

Reply:
Now that only [Windows 7+ is supported](https://doc.rust-lang.org/nightly/rustc/platform-support.html#tier-1-with-host-tools) the detailed version info seems redundant.